pom.xml

```diff
@@ -515,8 +515,8 @@
                 <configuration>
                     <skipCheckoutIfExists>false</skipCheckoutIfExists>
                     <connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
-                    <scmVersionType>tag</scmVersionType>
-                    <scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
+                    <scmVersionType>branch</scmVersionType>
+                    <scmVersion>CHK-891-refactoring-transaction-activation-requested-data-payment-context-code-dentro-notice-code</scmVersion>
```

Comment:
rollback this modification and update the `pagopa-ecommerce-commons.version`with the release version containing the counterpart modifications